plugins/cloud-aws/src/main/java/org/elasticsearch/cloud/aws/AwsEc2Service.java

```diff
@@ -50,10 +50,14 @@
     @Inject
     public AwsEc2Service(Settings settings, SettingsFilter settingsFilter, NetworkService networkService, DiscoveryNodeService discoveryNodeService) {
         super(settings);
+        // Filter global settings
         settingsFilter.addFilter("cloud.key");
         settingsFilter.addFilter("cloud.account");
         settingsFilter.addFilter("cloud.aws.access_key");
         settingsFilter.addFilter("cloud.aws.secret_key");
+        // Filter repository-specific settings
+        settingsFilter.addFilter("access_key");
```

Comment:
Does adding this automatically filter out the `cloud.aws.access_key`? Do we need both?